manifests/m/Microsoft/AzureCLI/2.42.0/Microsoft.AzureCLI.yaml

```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.42.0
+PackageName: Microsoft Azure CLI
+Publisher: Microsoft Corporation
+Moniker: azure-cli
+PackageUrl: https://docs.microsoft.com/en-us/cli/azure/
+License: Copyright (C) Microsoft Corporation
+ShortDescription: The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources. The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
+Installers:
+- Architecture: x64
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.42.0.msi
+  InstallerType: msi
+  InstallerSha256: b8ce13727e044efdf350e4d0bd49ce2aee418fcc6a1c10fe52ad046a5c97b521
+PackageLocale: en-US
+ManifestType: singleton
```

Comment:
Please move to the Multi-manifest format

Reply:
Hi @Trenly 
Is there any relevant doc or PR to guide how to use Multi-manifest format?

Reply:
https://learn.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Cversion-example#multiple-manifest-files

Using [wingetcreate](https://github.com/microsoft/winget-create) for your bot integrations or [CI/CD](https://github.com/microsoft/winget-create) will automatically use multi-manifest and latest ManifestVersion

Reply:
Hi @Trenly
I will add wingetcreate for our bot integrations later.
Please close this PR and move to #87136 which create by wingetcreate.